server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -287,4 +289,25 @@ public void test_kepserver_regclass_cast_query() throws Exception {
             ""
         ));
     }
+
+    @Test
+    public void test_pg_constrains_conkey_array_populated() throws Exception {
+        execute("CREATE TABLE doc.tbl (" +
+            "not_null int not null," +
+            "int_col int," +
+            "long_col bigint," +
+            "checked int CONSTRAINT positive CHECK(checked>0)," +
+            "PRIMARY KEY (int_col, long_col)," +
+            "CONSTRAINT many_cols_and_functions CHECK(not_null1 * long_col > int_col))"
+        );
+        int reloid = OidHash.relationOid(OidHash.Type.TABLE, new RelationName("doc", "tbl"));
+        response = execute("select i.conkey, i.conname, i.contype from pg_catalog.pg_constraint i where i.conrelid  = " + reloid + " order by i.conname");
+        assertThat(printedTable(response.rows()), is(
+            "[7]| check_and_not_null| c\n" +
+            "[5, 1, 2]| many_cols_and_functions| c\n" +
+            "[6]| positive| c\n" +
+            "[2, 3, 4]| tbl_pk| p\n"
```

Comment:
Are those assertions correct? the pk has only 2 columns, and the table in the test is not the same as the one in the `db-fiddle` test.

Reply:
I applied suggestion to simplify test but haven't pushed assertion fix yet)

Reply:
thx!